drs_filer/app_config.yaml

```diff
@@ -0,0 +1,50 @@
+# General server/service settings
+server:
+    host: '0.0.0.0'
+    port: 8080
+    debug: True
+    environment: development
+    testing: False
+    use_reloader: True
+
+#database
+db:
+    host: mongodb
+    port: 27017
+    dbs:
+        drsStore:
+            collections:
+                objects:
+                    indexes: null
+
+api:
+    specs:
+        - path: api/drsSpecs.yaml
+          append: null
+          add_operation_fields:
+              x-openapi-router-controller: ga4gh/drs/server
+          connexion:
+              strict_validation: True
+              validate_responses: False
+              options:
+                  swagger_ui: True
+                  serve_spec: True
+
+log:
+    version: 1
+    disable_existing_loggers: False
+    formatters:
+        standard:
+            class: logging.Formatter
+            style: "{"
+            format: "[{asctime}: {levelname:<8}] {message} [{name}]"
+    handlers:
+        console:
+            class: logging.StreamHandler
+            level: 20
+            formatter: standard
+            stream: ext://sys.stderr
+    root:
+        level: 10
+        handlers: [console]
+        
```

Comment:
Make sure to make use of FOCA's error handling (see comment below and the corresponding models and docs in FOCA for more infos).